samples/sqltransport/native-integration/sample.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SQL Server
+summary: 'How to integrate SQL Server transport with NHibernate persistence without outbox'
+tags:
+- SQL Server
+- NHibernate
+---
+
+ 1. Make sure you have SQL Server Express installed and accessible as `.\SQLEXPRESS`. Create a databases `samples`
+ 2. Start the Receiver project.
+ 3. In the Sender's console you should see `Press any key to send a message. Press `q` to quit` text when the app is ready. 
+ 4. Hit any key.
+ 5. A message will be sent using ADO.net and be received by the app.

```

Comment:
I'd separate the following steps from the one from 1 to 5, they are a different sample to run